pages/writing/scaling_support.mdx

```diff
@@ -0,0 +1,808 @@
+                <Button
+                  variant="primary"
+                  size="small"
+                  onClick={() => {
+                    setDisplayProp("flex");
+                  }}
+                >
+                  Emergency
+                </Button>
+              </Box>
+              <Box display={displayProp} marginTop="space40">
+                Aye aye! Paging a team member now. Hang tight!
+              </Box>
+            </Box>
+            <Text
+              as="div"
+              color="colorText"
+              fontSize="fontSize50"
+              marginTop="space10"
+              textAlign="right"
+            >
+              🤖
+            </Text>
+          </MediaBody>
+        </MediaObject>
+      </Box>
+      <Box display={displayProp} justifyContent="flex-end">
+        <MediaObject as="div">
+          <MediaBody>
+            <Box
+              as="div"
+              color="colorText"
+              backgroundColor="colorBackgroundUser"
+              borderRadius="borderRadius20"
+              borderBottomRightRadius="borderRadius0"
+              fontSize="fontSize30"
+              lineHeight="lineHeight20"
+              padding="space30"
+              maxWidth="size30"
+            >
+              Hi there, we intentionally haven't introduced classNames in Paste.
+              Read about it in our doc site!
+            </Box>
+            <Text
+              as="div"
+              color="colorText"
+              fontSize="fontSize50"
+              marginTop="space10"
+              textAlign="right"
+            >
+              🙂
+            </Text>
+          </MediaBody>
+        </MediaObject>
+      </Box>
+    </>
+  );
+};
+
+<Grid gutter="space40">
+  <Column span={4} offset={1}>
+    <Box>
+      <Stack orientation="horizontal" spacing="space30">
+        <Link href="/" passHref>
+          <Anchor>
+            <ScreenReaderOnly>Back to</ScreenReaderOnly> Home
+          </Anchor>
+        </Link>
+        <Text as="span">/</Text>
+        <Link href="/" passHref>
+          <Anchor>Writing</Anchor>
+        </Link>
+        <Text as="span">/</Text>
+      </Stack>
+    </Box>
+    <Heading as="h1" variant="heading10">
+      Growing Pains: How We{" "}
+      <Box as="span" backgroundColor="colorBackgroundBrandHighlight">
+        Scaled Our Design System Support
+      </Box>
+    </Heading>
+    <Paragraph>
+      While building platforms for the first time, your team and their customers
+      go through a multitude of changes. Customers learn new ways to work and
+      give up their own ways of working. And while most of these can be
+      positive, inertia and muscle memory pose challenges that might be
+      technical or operational, but almost always <strong>also</strong>{" "}
+      cultural. We saw this as our{" "}
+      <Link href="//paste.twilio.design/" passHref>
+        <Anchor>design system</Anchor>
+      </Link>{" "}
```

Comment:
This doesn't really look like a link, either because the words being linked are pretty generic, or the color.